README.md

```diff
@@ -32,7 +42,6 @@ An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
 From within the root directory:
 
 ```sh
-npm install -g webpack
 npm install
 ```
```

Comment:
Great job with the read me! Looks professional and  gives a user directions on how to start the program.